application/helpers/SurveyRuntimeHelper.php

```diff
@@ -417,7 +417,7 @@ public function run($surveyid, $args)
                 $aGroup['class'] = ' ls-hidden';
             }
 
-            $aGroup['name']        = $gl['group_name'];
+            $aGroup['name']        = LimeExpressionManager::ProcessString($gl['group_name'], null, null, 3, 1);
```

Comment:
You will need to strop {GROUPNAME} here.

Using the text " This is {GROUPNAME} inception" in the groupname leads to

"This is This is This is This is Group 2 inception inception inception inception "

and will probably create unnecessary load.

Reply:
Yes, but it's a Survey manager issue.
Still : easy fix : send only {GID}